torchvision/transforms/functional.py

```diff
@@ -1180,8 +1181,8 @@ def invert(img: Tensor) -> Tensor:
     Args:
         img (PIL Image or Tensor): Image to have its colors inverted.
             If img is a Tensor, it is expected to be in [..., H, W] format,
-            where ... means it can have an arbitrary number of trailing
-            dimensions.
+            where ... means it can have an arbitrary number of leading dimensions.
+            For PIL images, only mode "L" and "RGB" is supported.
```

Comment:
How about "F" mode ?

Reply:
It seems these ops all use the [PIL.ImageOps._lut()](https://pillow.readthedocs.io/en/stable/_modules/PIL/ImageOps.html?highlight=_lut#), and it only takes L or RGB at the moment.

Reply:
I'm not sure we currently have any tests for the specific mode or that there was intention to support them.